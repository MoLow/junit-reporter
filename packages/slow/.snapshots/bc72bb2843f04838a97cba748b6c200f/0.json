{
  "stderr": "",
  "stdout": "file: CWD/tests/slow_tests.js has slow tests:\n  *\n  *\n  *\n",
  "exitCode": 1
}