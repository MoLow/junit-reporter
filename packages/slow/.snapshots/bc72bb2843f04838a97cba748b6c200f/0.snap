Object {
  "exitCode": 1,
  "stderr": "",
  "stdout": "file: CWD/tests/slow_tests.js has slow tests:
  *
  *
  *
",
}